lib/state/actions/home_page_actions.dart

```diff
@@ -41,3 +36,18 @@ class GetEvolution extends LoadingAction {
     return state.copyWith(evolution: evolutionResponse);
   }
 }
+
+class GetFilterList extends LoadingAction {
+  static const key = 'get_filtered_list_key';
```

Comment:
not being used properly. There's no loading indication when this is called.

https://user-images.githubusercontent.com/47196695/182338288-b77c101c-8923-451c-b376-60c7cc6274e9.mov